index.html

```diff
@@ -0,0 +1,637 @@
+          </div>
+          <div class="col-lg-3 col-md-3 editors1">
+            <img src="images/s.png" alt="">
+            <a href="#">Grocery Stores Opening Early for Senior Citizens: Store Hours</a>
+            <p>Major grocery store chains like Walmart and Publix have adjusted their store hours, and some chains are opening early for senior citizens.</p>
+          </div>
+          <div class="col-lg-3 col-md-3 editors1">
+            <img src="images/t.png" alt="">
+            <a href="#">Coronavirus Can't Be Passed From Mothers to Babies During Labor: Study</a>
+            <p>There is no evidence the virus can be passed on during pregnancy or in breastmilk, according to The Royal College of Obstetricians and Gynaecologists.</p>
+          </div>
+
+        </div>
+
+      </div>
+
+    </section>
+
+    <section>
+      <h1>Hi</h1>
+      <div class="slideshow-heading">
+        <h4><img src="images/slide.png" alt=""> FEATURED SLIDESHOW</h4>
+        <hr>
+      </div>
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-4 slide1 col-md-4">
+            <img src="images/11.png" alt="">
+            <a href="#">Relive the 'Star Wars' Saga at These Filming Locations</a>
+          </div>
+          <div class="col-lg-4 slide1 col-md-4">
+            <img src="images/12.png" alt="">
+            <a href="#">Relive the 'Star Wars' Saga at These Filming Locations</a>
+
+          </div>
+          <div class="col-lg-4 slide1 col-md-4">
+            <img src="images/13.png" alt="">
+            <a href="#">Relive the 'Star Wars' Saga at These Filming Locations</a>
+          </div>
+          <div class="col-lg-4 slide1 col-md-4">
+            <img src="images/14.png" alt="">
+            <a href="#">Relive the 'Star Wars' Saga at These Filming Locations</a>
+          </div>
+          <div class="col-lg-4 slide1 col-md-4">
+            <img src="images/17.png" alt="">
+            <a href="#">Relive the 'Star Wars' Saga at These Filming Locations</a>
+
+          </div>
+          <div class="col-lg-4 slide1 col-md-4">
+            <img src="images/18.png" alt="">
+            <a href="#">Relive the 'Star Wars' Saga at These Filming Locations</a>
+
+          </div>
+
+        </div>
+
+      </div>
+    </section>
+
+    <section class="us">
+      <div class="container">
+        <div class="row">
+          <div class="us-heading">
+            <hr>
+            <h4>U.S</h4>
+          </div>
+          <div class=" col-lg-4 us-main col-md-5">
+            <img src="img/1.png" alt="">
+            <a href="#">Sanders Proposes $2,000 Monthly for Americans Until Coronavirus Threat Ends</a>
+          </div>
+          <div class="col-lg-4 us-main1 col-md-5">
+            <img src="img/2.png" alt="">
+            <a href="#">Conservative Author Warns Coronavirus 'Could Give Rise to the Antichrist'</a>
```

Comment:
- [ ] All these titles should be bold and have the proper hover effect. They repeat throughout the page.